sunpy/timeseries/tests/test_timeseriesbase.py

```diff
@@ -481,6 +522,20 @@ def test_concatenation_of_slices(eve_test_ts, concatenated_slices_test_ts):
     # ToDo: Will TSMD.concatenate() want to re-merge the metadata entries back into one?
 
 
+def test_concatenation_of_slices_list(eve_test_ts, concatenated_slices_test_list):
+    # Test resulting DataFrame is similar to the original
+    assert_frame_equal(concatenated_slices_test_list.to_dataframe(), eve_test_ts.to_dataframe())
+    # Otherwise: concatenated_ts.data.equals(eve_test_ts)
+    # Compare timeranges from before and after match for both metadata and TS
+    assert eve_test_ts.meta.time_range == concatenated_slices_test_list.meta.time_range
+    assert eve_test_ts.time_range == concatenated_slices_test_list.time_range
+    # Test metadata MetaDict matches
+    eve_test_ts.meta.metadata[0][
+        2] == concatenated_slices_test_list.meta.metadata[0][
+            2] == concatenated_slices_test_list.meta.metadata[1][2]
+    # ToDo: Will TSMD.concatenate() want to re-merge the metadata entries back into one?
```

Comment:
What do you mean by this?

Reply:
So I copied most of the code for that test from what was already present and that comment was part of what I copied over.

I dug around a bit and I think what that's asking is if we want `concatenate()` to combine the metadata of the concatenated TS objects.

Right now, say I have -

```python
>>> ts_1.meta.metadata = [(
        <sunpy.time.timerange.TimeRange object at 0x208ac7304c8>
        Start: 2016-09-30 05:01:00
        End:   2016-10-01 05:00:00
        Center:2016-09-30 17:00:30
        Duration:0.9993055555555556 days or
               23.983333333333334 hours or
               1439.0 minutes or
               86340.0 seconds,
        ["intensity"],
        MetaDict([("key", "value")])
    )]

>>> ts_2.meta.metadata = [(
        <sunpy.time.timerange.TimeRange object at 0x208a7683e48>
        Start: 2016-10-01 05:01:00
        End:   2016-10-02 05:00:00
        Center:2016-10-01 17:00:30
        Duration:0.9993055555555556 days or
               23.983333333333334 hours or
               1439.0 minutes or
               86340.0 seconds,
        ["intensity"],
        MetaDict([("key", "value")])
    )]
```

then `ts_1.concatenate(ts_2).meta.metadata` will simply give a list with both ts_1 and ts_2's metadata (`[*ts_1.meta.metadata, *ts_2.meta.metadata]`). This behavior carries over to when ts_2 is an iterable.

So instead of this, do we want to combine all of these metadata entries back into one or not.

Reply:
You mean to combine them into one metadata object?

Reply:
Yeah. It might make sense for when the resulting series is continuous. So we set Start and End to the minimum and maximum of all Start and all End times and sum up the Duration too. If the source is the same, the columns and relevant MetaDict are probably going to be the same too, so they can be combined as well. This is for a specific sub-case, but yes - we could add a kwarg like `combine` which could do this.

I'm curious if instantiating TimeSeries with `concatenate=True` does this already or not. I'll look into it.

Reply:
No, it doesn't do that. It just calls the `concatenate` method and it doesn't do that so invariably it doesn't either.